.github/workflows/ci.yml

```diff
@@ -473,20 +471,27 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest-xlarge]
+        os: [linux, windows, macos]
         interpreter: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12']
         include:
-          - os: windows-latest
+          # standard runners with override for macos arm
+          - os: linux
+            runs-on: ubuntu-latest
+          - os: windows
             ls: dir
+            runs-on: windows-latest
+          - os: macos
+            runs-on: macos-latest-xlarge
         exclude:
-          - os: macos-latest-xlarge
+          # macos arm only supported from 3.10 and up
```

Comment:
hi @davidhewitt! could you pls explain this comment? Thank you

Reply:
Yes that was purely limited by what we could download to run the build, IIRC.